action/command.py

```diff
@@ -29,8 +29,8 @@ class Command:
     Stores a Shell command with custom configuration options
     """
 
-    command: list[str] = field(default_factory=list)
-    expect: list[str] = field(default_factory=list)
+    command: str = field(default_factory=str)
```

Comment:
These lists of strings for command and expect were designed to allow the user to add a different response for a different output. If this feature is not needed, we can delete it.

Reply:
I know what it does, I just thought it's a confusing feature, and we're not using it anyway. I feel like this came from the fact that `pexpect` supports passing multiple expected things in a list. If we want to have some form of control flow, we should design it from the API first. Try and make it clear and easy to use, and then think about how to implement it.